### ISSUE TITLE
Armor glitches out.

### ISSUE THREAD

**Issue by dsweber2** [mchorse/emoticons#10]
Added emoticons to ftb revelations (v 1.12), and our armor was very spotty (mostly not there). The attached should have diamond pants and boots and [the mantle of the stars](https://ftb.gamepedia.com/Mantle_of_Stars). 
![image](https://user-images.githubusercontent.com/7095649/81224704-fb2e3100-8f9c-11ea-86d7-b63bd12e3443.png)

Thanks for this mod, would appreciate if you could look into this if you have the time

**Comment by dsweber2** [comment]
aand I should have checked closed issues https://github.com/mchorse/emoticons/issues/9. Might be good to leave it open, even if it is unfix-able, so people don't ask about it. Does it work with armor in the base game?

**Comment by mchorse** [comment]
Yeah, good idea, and yeah, default vanilla armor works (for exception of colored leather, I don’t think it was working last time I checked).
